### PR TITLE
[ANN-433][external] changed yolo format to conform to Darknet standard

### PR DESCRIPTION
**Pull request by tomvars**
Changed from `<class_id> <top_left_x_in_pixels> <top_left_y_in_pixels> <width_in_pixels> <height_in_pixels>` to `<class_id> <top_left_x_normalized> <top_left_y_normalized> <width_normalized> <height_normalized>` to conform with the Darknet YOLO standard as in https://pjreddie.com/darknet/yolo/

### CODE REVIEW

File: darwin/exporter/formats/yolo.py [hunk 1]
```diff
@@ -60,12 +60,22 @@ def _build_txt(annotation_file: dt.AnnotationFile, class_index: ClassIndex) -> s
...
 
         i = class_index[annotation.annotation_class.name]
-        x = round(data["x"])
-        y = round(data["y"])
-        w = round(data["w"])
-        h = round(data["h"])
+        # x, y should be the center of the box
+        # x, y, w, h are normalized to the image size
+        x = data["x"] + (data["w"] / 2)
+        y = data["y"] + (data["h"] / 2)
```
Comment (andreaazzini) [inline darwin/exporter/formats/yolo.py:70]:
```suggestion
        x = data["x"] + data["w"] / 2
        y = data["y"] + data["h"] / 2
```

File: setup.py [hunk 2]
```diff
@@ -36,8 +36,8 @@
         "pydantic",
     ],
     extras_require={
-        "test": ["responses", "pytest", "pytest-describe", "sklearn"],
-        "ml": ["sklearn", "torch", "torchvision"],
+        "test": ["responses", "pytest", "pytest-describe", "scikit-learn"],
+        "ml": ["scikit-learn", "torch", "torchvision"],
```
Comment (andreaazzini) [inline setup.py:40]: Is this necessary?

**Review by andreaazzini — CHANGES_REQUESTED** [review]

File: setup.py (hunk 2 above)
Comment (tomvars) [reply setup.py:40]: The import that was there is wrong see - https://pypi.org/project/sklearn/

Comment (tomvars) [reply setup.py:40]: Leads to failed tests if run at certain times of day.

**Review by andreaazzini — APPROVED** [review]

**Pull request merged**